system/ticketchallengescache.go

```diff
@@ -0,0 +1,61 @@
+package system
```

Comment:
Missing the decred developers. 

Also, I think all fuctions in this file can be made un-exported.